docs/testing-deployment-publishing/publishing/deploy-from-github.md

```diff
@@ -0,0 +1,74 @@
+---
+slug: /deployment/deploy-from-github
+title: Deploy from GitHub
+description: Learn how to deploy your apps directly from GitHub branch.
+tags: [Apple App Store, Google Play Store, Deployment, GitHub]
+sidebar_position: 5
+keywords: [Apple App Store, Google Play Store, Deployment, GitHub]
+---
+
+You can deploy your FlutterFlow app directly from a GitHub repository. This is especially useful when you are managing your app’s code on GitHub. Managing code on GitHub enables you to add custom features or advanced functionality that may not be fully supported by FlutterFlow's capabilities.
```

Comment:
should we mention first that if their flutterflow project is connected to a github repository, then the generated code can be pushed to the github repository and devs can manage deployment from the same repo instead of FF. 

I'm also not sure about this line: Managing code on GitHub enables you to add custom features or advanced functionality that may not be fully supported by FlutterFlow's capabilities.

like just mention about the added functionalities that can be added when the genrated code is managed in github and lets end there?

Reply:
ya, updated it. and removed the 'Managing code on GitHub enables you to ...' as the benefits are very well explained just below it.